src/phoenix/server/main.py

```diff
@@ -13,6 +13,7 @@
 import phoenix.trace.v1 as pb
 from phoenix.config import (
     EXPORT_DIR,
+    get_auth_settings,
```

Comment:
Can also just import the `ENABLE_AUTH` constant.

Reply:
I left it this way because you had some checks in there making sure both secret and enable is set?